### ISSUE TITLE
Event creation failed. The item could not be found.

### ISSUE THREAD

**Issue by Prozper** [phw198/OutlookGoogleCalendarSync#951]
Hi,

Im using OGCS a short while now but running into a problem:
We have a couple of Microsoft accounts imported to Google (gmail) with associated calendars.
We work in Outlook 2016 and use OGCS 2.8.3 to sync bidirectional ith push (and 120 min check).

All the agenda items sync, but a couple of meetings give an error (see pictures).
It seems to be happening with invitations sent by collegues.

Does anyone know what to do?

![Naamloos](https://user-images.githubusercontent.com/60396892/73279164-fd8a2d80-41ec-11ea-965d-d7805c8429ba.jpg)
![Knipsel](https://user-images.githubusercontent.com/60396892/73279732-dbdd7600-41ed-11ea-99b5-e4ff9e4b34be.PNG)

**Comment by phw198** [comment]
:spiral_notepad: Please increase logging level to `FINE`, reproduce the problem and upload your `OGcalsync.log` file ([see instructions](https://github.com/phw198/OutlookGoogleCalendarSync/wiki/Reporting-problems#locating-your-ogcalsynclog-file)).

**Comment by timotei82** [comment]
Same as me... 
[OGcalsync.log](https://github.com/phw198/OutlookGoogleCalendarSync/files/4191276/OGcalsync.log)

**Comment by phw198** [comment]
:link: #835 (contains debug level log)
```
2019-09-09 23:43:01,186 INFO  3 OutlookGoogleCalendarSync.Console [0] -  Creating 1 Google calendar entries
2019-09-09 23:43:01,439 DEBUG  3 OutlookGoogleCalendarSync.GoogleOgcs.Calendar [0] -  Processing >> 31/07/2019 15:00 => "FW: Lighthouse Sessions: Service Design in the Digital Era by Prof. Ivanka Visnjic from ESADE "
2019-09-09 23:43:01,440 DEBUG  3 OutlookGoogleCalendarSync.Console [0] -  31/07/2019 15:00  "FW: Lighthouse Sessions: Service Design in the Digital Era by Prof. Ivanka Visnjic from ESADE "
2019-09-09 23:43:01,530 DEBUG  3 OutlookGoogleCalendarSync.OutlookOgcs.OutlookNew [0] -  Retrieving the meeting organiser's timezone ID.
2019-09-09 23:43:01,780 ERROR  3 OutlookGoogleCalendarSync.Console [0] -  Event creation failed.
2019-09-09 23:43:01,780 ERROR  3 OutlookGoogleCalendarSync.Console [0] -  The item could not be found.
2019-09-09 23:43:01,835 ERROR  3 OutlookGoogleCalendarSync.OGCSexception [0] -  System.Runtime.InteropServices.COMException: The item could not be found.
2019-09-09 23:43:01,835 ERROR  3 OutlookGoogleCalendarSync.OGCSexception [0] -  Code: 0xA9340201;-1456209407
2019-09-09 23:43:01,835 ERROR  3 OutlookGoogleCalendarSync.OGCSexception [0] -     at Microsoft.Office.Interop.Outlook._ExchangeUser.get_PrimarySmtpAddress()
   at OutlookGoogleCalendarSync.OutlookOgcs.OutlookNew.GetRecipientEmail(Recipient recipient)
   at OutlookGoogleCalendarSync.GoogleOgcs.Calendar.CreateAttendee(Recipient recipient, Boolean isOrganiser)
   at OutlookGoogleCalendarSync.GoogleOgcs.Calendar.createCalendarEntry(AppointmentItem ai)
   at OutlookGoogleCalendarSync.GoogleOgcs.Calendar.CreateCalendarEntries(List`1 appointments)
```
:link: #722 Also the same error and both this and #835 issues are `Exchange connection mode: olNoExchange` - I'm guessing this issue here is too.

@Prozper Do each of the appointments with this error all say "The meeting was copied to your calendar..." when you open then, as per your screenshot?

**Comment by phw198** [comment]
:package: Please try again with [hotfix v2.8.3.5.zip](https://github.com/phw198/OutlookGoogleCalendarSync/files/4209000/v2.8.3.5.zip) ([see instructions for applying](https://github.com/phw198/OutlookGoogleCalendarSync/wiki/Reporting-Problems#applying-a-hotfix)).

It may not resolve it, but please let me know how it goes and upload your new `OGcalsync.log` file afterwards even if it does work.

Thanks!

@mgdmg

**Comment by phw198** [comment]
Believe this to be fixed under #1126.
